packages/lesswrong/server/codegen/typeGenerationUtils.ts

```diff
@@ -20,16 +20,26 @@ function maybeNullable(type: string, nullable: boolean) {
   return nullable ? `${type} | null` : type 
 }
 
-export function simplSchemaTypeToTypescript(schema, fieldName, simplSchemaType): string {
+export function simplSchemaTypeToTypescript(schema: SchemaType<DbObject>, fieldName: string, simplSchemaType, indent = 2): string {
```

Comment:
Leaving `simplSchemaType` untyped because none of the 3rd party type annotations we have access to via the `@types` package accurately describe this, and the code is sufficiently gnarly that I didn't want to spend a bunch of time coming up with my own.